From: =?utf-8?b?T25kxZllaiBOb3bDvQ==?= <onovy@debian.org>
Date: Wed, 3 Aug 2016 19:02:48 +0200
Subject: Use local objects.inv for intersphinx mapping

Forwarded: not-needed

Forwarded: not-needed
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- python-tornado.orig/docs/conf.py
+++ python-tornado/docs/conf.py
@@ -84,7 +84,7 @@
     )
 ]
 
-intersphinx_mapping = {"python": ("https://docs.python.org/3/", None)}
+intersphinx_mapping = {"python": ("/usr/share/doc/python3-doc/html", None)}
 
 html_theme = "sphinx_rtd_theme"
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
