From 87a85bb4de18ab17c360f7cb530274202fd05d4c Mon Sep 17 00:00:00 2001
From: Arnaud Schoonjans <arnaud.schoonjans@inmanta.com>
Date: Mon, 15 Dec 2025 12:12:18 +0100
Subject: [PATCH] Make sure that the in-operator on HTTPHeaders is case
 insensitive

--- python-tornado-6.4.2.orig/tornado/httputil.py
+++ python-tornado-6.4.2/tornado/httputil.py
@@ -223,7 +223,8 @@ class HTTPHeaders(collections.abc.Mutabl
         # in __getitem__ when it's not needed.
         if not isinstance(name, str):
             return False
-        return name in self._as_list
+        norm_name = _normalize_header(name)
+        return norm_name in self._as_list
 
     def __getitem__(self, name: str) -> str:
         header = _normalize_header(name)
--- python-tornado-6.4.2.orig/tornado/test/httputil_test.py
+++ python-tornado-6.4.2/tornado/test/httputil_test.py
@@ -312,6 +312,9 @@ Foo: even
             sorted(list(headers.get_all())),
             [("Asdf", "qwer zxcv"), ("Foo", "bar baz"), ("Foo", "even more lines")],
         )
+        # Verify case insensitivity in-operator
+        self.assertTrue("asdf" in headers)
+        self.assertTrue("Asdf" in headers)
 
     def test_malformed_continuation(self):
         # If the first line starts with whitespace, it's a
