Index: os-python-tornado/tornado/test/twisted_test.py
===================================================================
--- os-python-tornado.orig/tornado/test/twisted_test.py
+++ os-python-tornado/tornado/test/twisted_test.py
@@ -19,8 +19,7 @@ from tornado.testing import AsyncTestCas
 
 try:
     from twisted.internet.defer import (  # type: ignore
-        inlineCallbacks,
-        returnValue,
+        inlineCallbacks
     )
 
     have_twisted = True
@@ -43,7 +42,7 @@ class ConvertDeferredTest(AsyncTestCase)
                 # inlineCallbacks doesn't work with regular functions;
                 # must have a yield even if it's unreachable.
                 yield
-            returnValue(42)
+            return 42
 
         res = yield fn()
         self.assertEqual(res, 42)
